### DISCUSSION TITLE
Slack social link support

### DISCUSSION THREAD

**Discussion by russbiggs** [withastro/starlight#1071] [Feature Requests]
### What version of `starlight` are you using?

0.12.0

### What is your idea?

It would be nice to have `slack` available as an option for social links 

### Why is this feature necessary?

Though not a necessary feature, Starlight currently supports other chat platforms like discord. Some projects and communities use slack and would like to direct users to Slack for additional support and conversation

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Thanks for the suggestion, definitely a valid use case considering we have been adding more social links over time.

I see that you marked yourself as willing to submit a PR, is that still the case? If so, I think you can go ahead. If you need a reference regarding on how to add a new social link, you can check out the changes from [this PR](https://github.com/withastro/starlight/pull/892/files) which added Patreon support while ignoring the doc changes as this part is now automated.

And of course, if you need any help, feel free to ask.

**Reply by russbiggs** [reply]
Yup happy to contribute to help make this happy. Thanks for the example PR I'll follow that.

**Reply by russbiggs** [reply]
PR submitted, I'll keep an eye out if you need any further changes

**Comment by delucis** [comment]
Closing as this was released in v0.13 — thanks again @russbiggs!
